### PR TITLE
2.13 build

### PR DESCRIPTION
**Pull request by kubukoz**

### CODE REVIEW

File: core/src/test/scala/cats/effect/ConsoleSpec.scala [hunk 1]
```diff
@@ -89,7 +89,7 @@ class ConsoleSpec extends FunSuite {
         rs2 <- out2.get
         rs3 <- out3.get
       } yield {
-        assert(rs.right.get == List("foo", "undefined", "undefined"))
+        assert(rs.getOrElse(fail("Either.Left")) == List("foo", "undefined", "undefined"))
```
Comment (kubukoz) [inline core/src/test/scala/cats/effect/ConsoleSpec.scala]: `right` and `get` are deprecated.

File: project/Dependencies.scala [hunk 2]
```diff
@@ -3,14 +3,14 @@ import sbt._
 object Dependencies {
 
   object Versions {
-    val catsEffect = "1.3.1"
+    val catsEffect = "2.0.0-M4"
```
Comment (kubukoz) [inline project/Dependencies.scala]: Note: I'd advise against publishing a non-milestone with these changes :)

File: core/src/test/scala/cats/effect/ConsoleSpec.scala (hunk 1 above)
Comment (gvolpe) [reply core/src/test/scala/cats/effect/ConsoleSpec.scala]: Thank lord! :smile:

File: .sbtopts [hunk 3]
```diff
@@ -0,0 +1 @@
+-J-Xss2M
```
Comment (gvolpe) [inline .sbtopts]: Is this necessary?

Comment (kubukoz) [reply .sbtopts]: sbt was OOM: Metaspace-ing on me when running `+test`, but now that I think about it increasing stack size was placebo (solves different sbt issues though)... will revert this one

Comment (kubukoz) [reply .sbtopts]: Removed

**Comment by kubukoz** [comment]
Should be ready for a milestone now :)

**Pull request merged**